content/blog/2025/kai-release-announcement/index.md

```diff
@@ -0,0 +1,62 @@
+
+After months of ideas, development, and feedback from early users and contributors, we're excited to share the first official release of **[Konveyor AI 0.1.0](https://github.com/konveyor/editor-extensions/releases/tag/v0.1.0)**.
+
+This release is a significant step forward for the Konveyor project, and for anyone working through the complex journey of application modernization. It marks the beginning of **developer-focused, AI-assisted modernization** built with community input and open collaboration.
+
+## What is Konveyor AI?
+
+**Konveyor AI (Kai)** provides real-time insights to help developers and migrators identify and refactor deprecated or outdated code. As a developer modernizes an application with Kai, they'll see detailed issues and their location, solutions with the reasoning for it, and code suggestions that can be automatically implemented.
+
+Built on top of the Konveyor ecosystem, Kai brings:
+
+- **Static code analysis**:
+  Identifies potential issues when migrating to  new technologies (e.g., updating Java frameworks, containerizing an application).
+
+- **Extensible migration paths**:
+  Includes 2,400 predefined rules for different migration paths. Users can create their own custom rules for unique scenarios. 
+
+- **VS Code extension**: 
+  Suggested code changes show up within the IDE.
+
+- **Model-agnostic AI**:  
+  Users are not locked into a specific LLM, enabling flexibility in AI-driven modernization
+
+- **Agentic AI for intelligent responses**:  
+  Delivers more meaningful, sanitized suggestions, validating Maven compilations and dependency resolutions.
+
+Kai can assist you with migrating from legacy Java, upgrading to Spring Boot 3, or prepping for a cloud native deployment. You can also [build custom rulesets](https://github.com/konveyor/analyzer-lsp/blob/main/docs/rules.md) when needed.
+
+## Try it out!
+
+The 0.1.0 release is available [here](https://github.com/konveyor/editor-extensions/releases/download/v0.1.0/konveyor-v0.1.0.vsix)!
+
+You can take Kai for a spin using [step-by-step guided scenarios](https://github.com/konveyor/kai/tree/main/docs/scenarios).
+
+If you are interested in learning how Kai works, here are some links to the technical blogs,
+
+- [**Kai - Generative AI Applied to Application Modernization**](https://www.cncf.io/blog/2024/11/22/konveyor-ai-supporting-application-modernization/)
+- [**Incident Storage in Kai - A Deep Dive**](https://konveyor.io/blog/2023/one-year-cncf-sandbox-konveyor-reflective-journey/)
+- [**Konveyor AI: Supporting Application Modernization**](https://www.cncf.io/blog/2024/11/22/konveyor-ai-supporting-application-modernization/)
+
+
+We would like to thank the Konveyor community for their continued support, feedback, and encouragement. Huge shoutout to the contributors ([IDE extension](https://github.com/konveyor/editor-extensions/graphs/contributors), [backend](https://github.com/konveyor/kai/graphs/contributors)) who made this release possible. Thank you!
```

Comment:
Needs revision, this is the Konveyor community, so who's "we"?

Reply:
updated 'we' -> 'Konveyor maintainers'